<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>208</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Log into Virtual World</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>First and Last Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_fullNameCombo">
       <property name="toolTip">
        <string>Enter your first and last name, separated by a space.</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtCurrent</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_passwordEdit">
       <property name="toolTip">
        <string>Enter your password.</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>SIM Landmark</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="m_slurlEdit">
       <property name="toolTip">
        <string>Select where in the grid you want to appear. If unsure, leave it set to &quot;Home.&quot;
SIM Landmarks are in the form &quot;SIM/X/Y/Z&quot;. For example: &quot;Dolce Vita/245/100/50&quot;.</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Home</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/mainIcon/home_icon</normaloff>:/mainIcon/home_icon</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>My Last Location</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/mainIcon/main_icon</normaloff>:/mainIcon/main_icon</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Grid</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="m_gridEdit">
       <property name="toolTip">
        <string>Select the grid to connect to. If you don't know what this is, better to leave it on &quot;second life.&quot;</string>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="m_savePasswordCheckBox">
       <property name="toolTip">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;SLiteChat can remember your password for you as a convenience.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Do NOT check this if you are using a public computer&lt;/span&gt;!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Save Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_resetFullNames">
       <property name="text">
        <string>&amp;Reset Names</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_resetSims">
       <property name="toolTip">
        <string>Click this to clear out your landmark entries.</string>
       </property>
       <property name="text">
        <string>Reset &amp;Landmarks</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="m_buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_fullNameCombo</tabstop>
  <tabstop>m_passwordEdit</tabstop>
  <tabstop>m_slurlEdit</tabstop>
  <tabstop>m_gridEdit</tabstop>
  <tabstop>m_savePasswordCheckBox</tabstop>
  <tabstop>m_resetFullNames</tabstop>
  <tabstop>m_resetSims</tabstop>
  <tabstop>m_buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LoginWindow</receiver>
   <slot>OnAccepted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LoginWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_passwordEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>LoginWindow</receiver>
   <slot>OnPasswordChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_savePasswordCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>LoginWindow</receiver>
   <slot>OnSavePasswordChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_resetSims</sender>
   <signal>clicked()</signal>
   <receiver>LoginWindow</receiver>
   <slot>OnResetSimLandmarks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_gridEdit</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>LoginWindow</receiver>
   <slot>OnGridChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fullNameCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>LoginWindow</receiver>
   <slot>OnFullNameChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fullNameCombo</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>LoginWindow</receiver>
   <slot>OnFullNameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_resetFullNames</sender>
   <signal>clicked()</signal>
   <receiver>LoginWindow</receiver>
   <slot>OnResetFullNames()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnTextChanged(QString)</slot>
  <slot>OnSavePasswordChecked(bool)</slot>
  <slot>OnAccepted()</slot>
  <slot>OnHomeChecked(bool)</slot>
  <slot>OnResetSimLandmarks()</slot>
  <slot>OnGridChanged(int)</slot>
  <slot>OnFullNameChanged(int)</slot>
  <slot>OnResetFullNames()</slot>
  <slot>OnPasswordChanged(QString)</slot>
  <slot>OnFullNameChanged(QString)</slot>
 </slots>
</ui>
